### PR TITLE
update offers and webhooks

### PR DESCRIPTION
**Pull request by andrelias**
Updated offers documentation and added offer activation. Also added a note for qualification webhooks.

### CODE REVIEW

File: offers/README.md [hunk 1]
```diff
@@ -246,126 +217,104 @@ fileName:offer.json
...
-### Active
-Offer goes active on `startDate` and starts qualifying transactions at the participating locations.
-  
-### Rejected
-If an offer is rejected by a publisher, you receive a notification email with the rejection reason and you can to edit the offer parameters and resubmit the offer. 
+If the offer is labeled as pending it means that the `startDate` is in the future. When an offer has at least one location linked to it and the `startDate` is set in the future, the offer is placed in the all-set group. An offer is live when today's date is between the offer's `startDate` and `endDate` and `locationsTotal` is greater than `0`.
 
 ### Pending
-Offer was created with no publisher. These offers are advertised to publishers on the Fidel CLO marketplace.
+Offer `startDate` is set in the future. Offer will start qualifying transactions made at locations linked to the offer after the `startDate`.
```
Comment (c0derabbit) [inline offers/README.md:259]: I think some words are missing here, don't really understand this sentence. maybe `Offer will start qualifying when transactions are made`?

File: offers/README.md [hunk 2]
```diff
@@ -9,41 +9,40 @@ A card linked offer specifies a set of parameters that will be used to qualify a
...
-  "startDate": "2018-10-19T00:00:00.000Z",
-  "status": "ACTIVE",
+  "startDate": "2020-04-20T00:00:00.000Z",
   "type": {
     "name": "discount",
     "value": 20
   },
-  "updated": "2018-10-19T12:12:00.000Z",
-  "userId": "82c5a9ed-5301-46ab-8599-6bcb0d017cc6"
+  "updated": "2018-10-19T12:12:00.000Z"
```
Comment (diogogvhenriques) [inline offers/README.md:45]: small thing here could this date be closer to start date? :)

File: offers/README.md [hunk 3]
```diff
@@ -246,126 +217,104 @@ fileName:offer.json
...
 
-### Syncing
-Offer has been accepted by the publisher and participating locations are now being onboarded with the schemes.  This stage takes between 1 and 10 business days.
-
-### Active
-Offer goes active on `startDate` and starts qualifying transactions at the participating locations.
-  
-### Rejected
-If an offer is rejected by a publisher, you receive a notification email with the rejection reason and you can to edit the offer parameters and resubmit the offer. 
+If the offer is labeled as pending it means that the `startDate` is in the future. When an offer has at least one location linked to it and the `startDate` is set in the future, the offer is placed in the all-set group. An offer is live when today's date is between the offer's `startDate` and `endDate` and `locationsTotal` is greater than `0`.
```
Comment (joao8lopes) [inline offers/README.md:256]: Having an offer labeled as `pending` doesn't necessarily mean that the `startDate` is in the future! The `startDate` can be "now" but the offer doesn't have a location linked or a funder set.

File: offers/README.md [hunk 4]
```diff
@@ -246,126 +217,104 @@ fileName:offer.json
...
+### Expired
+Current day is after the `endDate`. The offer has stopped qualifying transactions.
 
 <hr>
 
 ## Qualification
 
-When an offer is `Active`, the transaction qualification will start. Every transaction made a by a linked card on a location where an offer is active will be analysed considering the offer parameters and can qualify or not qualify for the offer.
-In both cases, an offer object is appended to the original transaction object containing all the qualification offer data. In case the transaction qualifies, `cashback` and `performanceFee` amounts are automatically calculated and the `qualified` property is set to `true`. If the transactions does not qualify, `cashback` and `performanceFee` amounts will be `0` and `qualified` property `null`.
+When an offer is live, each transaction made a by a linked card on a location linked to the offer will be analysed against the offer parameters and can qualify or not qualify for the offer.
```
Comment (diogogvhenriques) [inline offers/README.md:271]: > When an offer is live, each transaction made by a linked card

remove the extra 'a'

**Pull request merged**